packages/vscode-extension/src/treeview/account/accountTreeViewProvider.ts

```diff
@@ -30,19 +31,11 @@ export class AccountTreeViewProvider implements vscode.TreeDataProvider<DynamicN
   }
 
   public subscribeToStatusChanges(tokenProvider: TokenProvider) {
-    tokenProvider.appStudioToken?.setStatusChangeMap("tree-view", (status, token, accountInfo) =>
-      m365AccountStatusChangeHandler("appStudio", status, token, accountInfo)
-    );
-    // TODO: remove after m365 account providers are unified
-    tokenProvider.sharepointTokenProvider?.setStatusChangeMap(
-      "tree-view",
-      (status, token, accountInfo) =>
-        m365AccountStatusChangeHandler("sharepoint", status, token, accountInfo)
-    );
-    tokenProvider.graphTokenProvider?.setStatusChangeMap(
+    tokenProvider.m365TokenProvider?.setStatusChangeMap(
       "tree-view",
+      { scopes: AppStudioScopes },
       (status, token, accountInfo) =>
-        m365AccountStatusChangeHandler("graph", status, token, accountInfo)
+        m365AccountStatusChangeHandler("appStudio", status, token, accountInfo)
```

Comment:
Can this `token` returned from m365 provider be used to check sideloading status? If so, we could remove the `source` parameter of `m365AccountStatusChangeHandler` because it was used to know which token and only "appStudio" token worked. Thanks!

Reply:
If we configurate `{ scopes: AppStudioScopes }` in the setStatusChangeMap. We will get the right token for sideloading check. Yes, we can remove `source` parameter.